src/back/wgsl/writer.rs

```diff
@@ -173,6 +182,30 @@ impl<W: Write> Writer<W> {
         Ok(())
     }
 
+    /// Helper method used to write stuct name
+    ///
+    /// # Notes
+    /// Adds no trailing or leading whitespace
+    fn write_struct_name(&mut self, module: &Module, handle: Handle<Type>) -> BackendResult {
+        if module.types[handle].name.is_none() {
+            #[allow(clippy::pattern_type_mismatch)]
```

Comment:
please no :)

Reply:
I just think it's a false positive clippy suggestion here. Sorry, my Rust-fu not so good :) Thanks a lot for your patience!